Tests/Unit/DependencyInjection/CmfSeoExtensionTest.php

```diff
@@ -368,8 +368,8 @@ public function testDefaultTemplatesSet()
             array(
                 'sitemap' => array(
                     'templates' => array(
-                        'xml' => CmfSeoBundle::DEFAULT_SITEMAP_TEMPLATE_XML,
-                        'html' => CmfSeoBundle::DEFAULT_SITEMAP_TEMPLATE_HTML,
```

Comment:
tests should not rely on such a default value or you won't detect accidental change. and changing that template name would be a BC break for people that overwrote the template rather than configuring one explicitly.